openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -654,6 +654,88 @@
       display: block;
     }
 
+    #openassessment_rubric_clone {
+      @include rubric-reuse-jquery-ui;    
+      .ui-accordion-header {
+        padding: 5px;
+        font-weight: 500;
+        font-size: 18px;
+        line-height: 28px;
+
+        #rubric-reuse-collapse-icon::after {
+          @include float(right);
+          font-family: FontAwesome;
+          display: inline-block;
+          color: inherit;
+          padding: 0 5px;
+          content: "\f067";
+        }
+      }
+
+      .ui-accordion-header-active #rubric-reuse-collapse-icon::after {
+        content: "\f068";
+      }
+
+      .ui-accordion-header-icon {
+        display: none
+      }
+
+      .openassessment_rubric_clone_block{
+        padding: 8px 0px;
+      }
+
+      .ui-accordion-content {
+        overflow: visible;
+        
+        #openassessment_rubric_clone_form_container {
+          display: flex;
+
+          #openassessment_rubric_clone_input_container {
+            flex: 9;
+
+            #openassessment_rubric_clone_input {
+              padding: 6px;
+              width: 90%;
+              height: 40px;
+            }
+
+            .openassessment_clone_combobox {
+              position: relative;
+              display: inline-block;
+            }
+
+            .openassessment_clone_combobox_toggle {
```

Comment:
Fix for weird button styling:

```suggestion
            .openassessment_clone_combobox_toggle {
              vertical-align: middle;
```

Reply:
If this one little fix is all it takes .... I love css! /s

Reply:
un
![Screen Shot 2021-07-23 at 1 55 37 PM](https://user-images.githubusercontent.com/1639231/126822875-8768998e-e3bc-4f44-b418-63ed668edef8.png)
fortunately this didn't fix it

Reply:
It's something to do with the padding around the text input i think? I'm really not sure though